docs/sdk/api/sdkMiddlewareAuth.md

```diff
@@ -151,29 +151,30 @@ const client = createClient({
 })
 ```
 
-## `createAuthMiddlewareForExistingToken(options)`
+## `createAuthMiddlewareWithExistingToken(options)`
 
-Creates a [middleware](/sdk/Glossary.md#middleware) that attaches a known access token a request header.
+Creates a [middleware](/sdk/Glossary.md#middleware) that attaches a known access token `Authorization` header.
 
-#### Named arguments (options)
+#### Named arguments (authorization, options)
+
+`authorization` _(String)_: the Authorization to attach to the request header. Can be in the format `"Bearer myAccessToken"`
 
-`options` can be a string or an object. If `options` is a _(String)_, it is treated as the access token with other possible options set to their defaults. If `options` is an _(Object)_, it can have the following properties:
+`options` is an optional _(Object)_, having the following properties:
 
-1. `token` _(String)_: the access token to be attached to the request
-2. `tokenType` _(String)_: the type of the access token (Default: `Bearer`)
-3. `force` _(Boolean)_: if set to true, existing Authorization header (if any) in the request will be overridden with the supplied access token (Default: `true`)
+1. `force` _(Boolean)_: if set to true, existing Authorization header (if any) in the request will be overridden with the supplied access token (Default: `true`)
 
 ```js
 import { createClient } from '@commercetools/sdk-client'
-import { createAuthMiddlewareForExistingToken } from '@commercetools/sdk-middleware-auth'
+import { createAuthMiddlewareWithExistingToken } from '@commercetools/sdk-middleware-auth'
+
+const accessToken = 'my-access-token'
 
 const client = createClient({
   middlewares: [
-    createAuthMiddlewareForExistingToken({
-      token: 'my-access-token',
-      tokenType: 'Bearer',
-      force: true,
-    }),
+    createAuthMiddlewareWithExistingToken(
+      `Bearer ${accessToken}`,
+      { force: true }
```

Comment:
Ah so now we "only" work with bearer tokens.

Reply:
No, it's just an example. It's up to the user to define the auth scheme.

Reply:
Ah ok. Sorry missed that.